component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -812,15 +807,18 @@ private void processValidUserToken(AuthenticationContext context, AuthenticatedU
 
     /**
      * If user forgets the mobile, then user can use the back up codes to authenticate the user.
+     * Check whether the entered code matches with a backup code.
      *
-     * @param context           the AuthenticationContext
-     * @param userToken         the userToken
-     * @param authenticatedUser the name of authenticatedUser
-     * @throws AuthenticationFailedException
+     * @param context           The AuthenticationContext.
+     * @param userToken         The userToken.
+     * @param authenticatedUser The authenticatedUser.
+     * @return True if the user entered code matches with a backup code.
+     * @throws AuthenticationFailedException Authentication failure exception when retrieving user claim for OTP list.
```

Comment:
Shall we update this as follows? 
`If an error occurred while retrieving user claim for OTP list.`
WDYT?

Reply:
Fixed in a8b29a5